### ISSUE TITLE
CDC does not sink data to destination tikv cluster 

### ISSUE THREAD

**Issue by haojinming** [tikv/migration#161]
### 1. Describe the bug
CDC does not sink data to destination tikv cluster if there are much(~800G) data when creating changefeed.

### 2. Minimal reproduce step (Required)
1. Deploy tikv cluster with tikv-cdc
2. Change tidb gc lifetime to 72h, `update mysql.tidb set variable_value='72h' where variable_name='tikv_gc_life_time';`
3. Load ~800G data into tikv cluster
4. Create changefeed to sink data to another tikv cluster.

### 3. What did you see instead (Required)
No data is sink to destination cluster. 
Clinic data is : https://clinic.pingcap.com.cn/portal/#/orgs/293/clusters/7121692355614174507?from=1658151085&to=1658194285
### 4. What did you expect to see? (Required)
cdc can handle ~800g data the sink data to destination cluster.
### 5. What is your migration tool and TiKV version? (Required)

- TiKV CDC:

**Comment by pingyu** [comment]
The issue is caused by OOM.
```
[699912.727886] Out of memory: Kill process 14895 (cdc) score 946 or sacrifice child
[699912.731437] Killed process 14895 (cdc), UID 0, total-vm:32728536kB, anon-rss:31913920kB, file-rss:440kB, shmem-rss:0kB
```

**Comment by haojinming** [comment]
This issue is caused by OOM and has been fixed in latest release [cdc-v1.1.0](https://github.com/tikv/migration/releases/tag/cdc-v1.1.0). Flow control is added in https://github.com/tikv/migration/pull/242 .
